sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/datastore/V1Beta3TestOptions.java

```diff
@@ -0,0 +1,26 @@
+package org.apache.beam.sdk.io.gcp.datastore;
+
+import org.apache.beam.sdk.options.Description;
+import org.apache.beam.sdk.options.Validation;
+import org.apache.beam.sdk.testing.TestPipelineOptions;
+
+import javax.annotation.Nullable;
+
+/**
+ * V1Beta3 Datastore related pipeline options.
+ */
+public interface V1Beta3TestOptions extends TestPipelineOptions {
```

Comment:
+ @jasonkuster

Are we supposed to extend `TestPipelineOptions` directly vs map TestPipelienOptions into local test options? I thought the latter.